defaults/syscalls.go

```diff
@@ -0,0 +1,337 @@
+package defaults
+
+// Syscall list for Seccomp rules
+const (
+	SysRead                = "read"
```

Comment:
instead of typing these as golang strings, could we define a custom type that just wraps strings?
  Can be `syscall` or similar